HISTORY.md

```diff
@@ -1,3 +1,16 @@
+4.1.0 / 2019-11-04
+
+  * deps: handlebars@4.5.1
+    - Add method Handlebars.parseWithoutProcessing
+    - Add guard to if & unless helpers
```

Comment:
It's not clear what this would mean to users of `hbs`. Is this just an internal change to `handebars` that users cannot see? If so, you don't need to list it, if not, please expand on what the change means